src/mixins/input.js

```diff
@@ -21,7 +21,18 @@ export default {
   props: {
     appendIcon: String,
     appendIconCb: Function,
-    asyncLoading: Boolean,
+    asyncLoading: {
+      type: [Boolean, Number],
+      default: false
+    },
+    asyncLoadingHeight: {
+      type: [String, Number],
+      default: 2
+    },
+    asyncLoadingColor: {
```

Comment:
I would rather expose the progress spot through a scoped slot instead of adding tons of props to mirror to it.